docs/intro/tutorial03.txt

```diff
@@ -449,6 +449,7 @@ file, go ahead and add an ``app_name`` to set the application namespace:
     :filename: polls/urls.py
 
     from django.conf.urls import url
+    from . import views
```

Comment:
do I have to leave a blank line above the import line ?

On Sat, Jan 2, 2016 at 3:37 PM, Tim Graham <notifications@github.com> wrote:

> In docs/intro/tutorial03.txt
> <https://github.com/django/django/pull/5914#discussion_r48683853>:
>
> > @@ -449,6 +449,7 @@ file, go ahead and add an ``app_name`` to set the application namespace:
> >      :filename: polls/urls.py
> >
> >      from django.conf.urls import url
> > +    from . import views
>
> If you want to try amending this pull request, you could add a newline
> above this line for consistency and amend the commit message to "Added a
> missing import to tutorial 3." If not, it's easy enough to make these
> updates while merging. Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/django/django/pull/5914/files#r48683853>.
>



-- 
Regards,
Mahendra Yadav